app/views/home/index.html.erb

```diff
@@ -1,4 +1,34 @@
 <div class="container">
   <h1>Welcome to Imposter!</h1>
-  <h2>Coming Soon</h2>
-</div>
\ No newline at end of file
+  
+  <div class="rave-controls">
+    <table class="rave-table">
+      <tr>
+        <th>
+        
+        <div class="rave-status">
+          <% if @status == "on" %>
+            <p class="rave-on">ON</p>
```

Comment:
I was able to get rid of it, you can both see if you like it or not. I switched the second conditional I had in this view to a ternary op as well, looks WAY better